app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/AddProductNutritionFactsData.java

```diff
@@ -88,6 +88,9 @@
         PREFIX_NUTRIMENT_LONG_NAME + "vitamin-k",
         PREFIX_NUTRIMENT_LONG_NAME + "zinc"));
 
+    private AddProductNutritionFactsData() {
```

Comment:
I'm not sure about this one. Are you sure we don't need to initialize this fragment anywhere else?

Reply:
I think this might be a misplaced file. It's not actually a `Fragment`, but a utility class that's used by the `AddProductNutritionFactsFragment` class.